src/main/java/core/basesyntax/service/impl/DeepCopyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
```

Comment:
It seems that the `FruitTransaction` class is located in the `db` package, which suggests it is a database entity. Generally, database entities should not be directly used for other purposes like DTOs (Data Transfer Objects). Consider creating a separate DTO class if needed for deep copying.